src/JustEat.StatsD/Buffered/Tags/IStatsDTagsFormatter.cs

```diff
@@ -2,12 +2,32 @@
 
 namespace JustEat.StatsD.Buffered.Tags
 {
-    internal interface IStatsDTagsFormatter
```

Comment:
I don't think we'd want the ref struct to be in the public interface. Is there a way to expose this interface using strings in a simpler way, and then have an adapter internally like we have today that maps the user-specified one to the internal one that uses buffers?

Reply:
Sure. I will change the `IStatsDTagsFormatter` to return a string and add it to the buffer internally.